src/modules/peek/Peek.FilePreviewer/Previewers/MediaPreviewer/ImagePreviewer.cs

```diff
@@ -199,74 +207,5 @@ await Dispatcher.RunOnUiThread(async () =>
                 });
             });
         }
-
-        private static readonly HashSet<string> _supportedFileTypes = new HashSet<string>
-        {
-                // Image types
-                ".bmp",
-                ".gif",
-                ".jpg",
-                ".jfif",
-                ".jfi",
-                ".jif",
-                ".jpeg",
-                ".jpe",
-                ".png",
-                ".tif",  // very slow for large files: no thumbnail?
-                ".tiff", // NEED TO TEST
-                ".dib",  // NEED TO TEST
-                ".heic",
-                ".heif",
-                ".hif",  // NEED TO TEST
-                ".avif", // NEED TO TEST
-                ".jxr",
-                ".wdp",
-                ".ico",  // NEED TO TEST
-                ".thumb", // NEED TO TEST
-                ".webp",
-
-                // Raw types
-                ".arw",
-                ".cr2",
-                ".crw",
-                ".erf",
-                ".kdc", // NEED TO TEST
-                ".mrw",
-                ".nef",
-                ".nrw",
-                ".orf",
-                ".pef",
-                ".raf",
-                ".raw",
-                ".rw2",
-                ".rwl",
-                ".sr2",
-                ".srw",
-                ".srf",
-                ".dcs", // NEED TO TEST
-                ".dcr",
-                ".drf", // NEED TO TEST
-                ".k25",
-                ".3fr",
-                ".ari", // NEED TO TEST
-                ".bay", // NEED TO TEST
-                ".cap", // NEED TO TEST
-                ".iiq",
-                ".eip", // NEED TO TEST
-                ".fff",
-                ".mef",
-
-                // ".mdc", // Crashes in GetFullBitmapFromPathAsync
```

Comment:
Wonder if we should exclude this one as well... Did you test it? Couldn't find an accessable file to give it a test.

Reply:
Hi @jaimecbernardo !

On my Windows 11 machine, ".mdc" files are not supported by `BitmapDecoder` / WIC, so this new version of Peek still won't attempt to load them via ImagePreviewer. The files' basic properties will be shown by the UnsupportedFilePreviewer instead.

I'm not sure why it was in the list originally then commented out later. I wonder if the author had the Minolta RAW plugin registered and that lit up support on their machine, but that's just a guess.

To answer your question: I've downloaded a bunch of RAW files over the last few months, but I don't have a complete set to match the full list of formats supported by Windows. (Perhaps downloading and testing them all would be a good exercise for a rainy day!).  https://www.rawsamples.ch/ has some MDC file downloads, but they've never rendered for me because of them not being natively supported. I haven't had any issues with RAW images I tried with extensions which `BitmapDecoder` reports as compatible.

Long story short: we don't need to explicitly exclude ".mdc" from the list!